src/app.styled.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export const MainContainer = styled('div')`
```

Comment:
Use `main`, please

It is a good practice for Search Engines and screen readers to use semantic HTML tags.

Take a look [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html)

And [there is](https://web.dev/learn/accessibility/) also a good accessibility course by Google developers